src/js/editor.js

```diff
@@ -485,13 +495,19 @@ document.addEventListener('mouseover', (event) => {
 
   if (className !== 'CodeMirror-linenumber') return
 
-  const activeLine = cm.getCursor().line + 1
+  const line = cm.getCursor().line
+  const activeLine = line + 1
   const isValid = activeLine > +event.target.innerText
+  const hasError = event.target.parentElement.classList.contains('lineNoError')
```

Comment:
could be `CLASS_LINE_ERROR` i think

Reply:
`CLASS_LINE_ERROR` doesn't exist in editor.js (It's used in eval.js).  I can add it as a const there but not sure if it's really needed for this.

Reply:
ah, makes sense